livekit-agents/livekit/agents/utils/codecs/mp3.py

```diff
@@ -14,9 +14,9 @@
 
 import ctypes
 import logging
-from importlib import import_module
 from typing import List
 
+import av
```

Comment:
av is an optional dependency, if we load it here, it'll raise Exception for those who doesn't want it